packages/cssnano/src/index.js

```diff
@@ -76,7 +77,12 @@ function resolveConfig(options) {
     configPath = path.resolve(process.cwd(), options.configFile);
   }
 
-  const configExplorer = cosmiconfigSync(cssnano);
+  const configExplorer = lilconfigSync(cssnano, {
+    loaders: {
+      '.yaml': (filepath, content) => yaml.parse(content),
+      '.yml': (filepath, content) => yaml.parse(content),
+    },
```

Comment:
I wonder if you may also need to add something like `searchPlaces: ['.cssnanorc.yaml', '.cssnanorc.yml']` to the options in order for lilconfig to actually look for yaml files?

It seems like lilconfig uses this as basis for [merging in the loaders](https://github.com/antonk52/lilconfig/blob/c54ebad1a7916c764a423c8221013c63b2d1ce63/src/index.ts#L105).

Ref. https://github.com/cssnano/cssnano/pull/1168#discussion_r671873676

Reply:
Seems like a bit of a bug or design weakness on their part

Reply:
I've updated the code with explicit search places. I did try renaming the file to `.cssnanorc.yml` and it worked.

Reply:
Confirmed! And perhaps not such a bad thing for it to be explicit anyway 👍